napari/settings/_updates.py

```diff
@@ -0,0 +1,54 @@
+from typing import List
+
+from pydantic import Field
+
+from ..utils.events.evented_model import EventedModel
+from ..utils.misc import is_dev
+from ..utils.translations import trans
+
+
+class UpdateSettings(EventedModel):
+    check_for_updates: bool = Field(
+        True,
+        title=trans._("Automatically check for napari updates"),
+        description=trans._("Automatically check for napari updates."),
+    )
+    update_automatically: bool = Field(
+        False,
+        title=trans._("Update to the latest version automatically"),
+        description=trans._("Update to the latest version automatically."),
+    )
+    check_previews: bool = Field(
+        False,
+        title=trans._("Check preview candidates "),
```

Comment:
Might good to have a tooltip for this too e.g. "Check for next release candidate" or something like that

Reply:
The description is the thing that acts as tooltip for the settings. 

Also these options are hidden for this version, while we decide how to best surface them to users (they were shown in dev mode)